src/common/pool_map.c

```diff
@@ -1523,52 +1547,49 @@ int
 pool_map_create(struct pool_buf *buf, uint32_t version, struct pool_map **mapp)
 {
 	struct pool_domain *tree = NULL;
-	struct pool_map	   *map = NULL;
+	struct pool_map	   *map;
 	int		    rc;
 
 	rc = pool_buf_parse(buf, &tree);
 	if (rc != 0) {
-		D_ERROR("pool_buf_parse failed, rc "DF_RC"\n", DP_RC(rc));
+		D_ERROR("pool_buf_parse failed: "DF_RC"\n", DP_RC(rc));
```

Comment:
Minor, but this is a very common thing to log so I wonder if we should have a macro that just takes function name and RC.

Mostly I've been doing "function() failed, "DF_RC"\n", DP_RC(rc) but there are cases with , or : or neither and some say "rc =" where some just print the RC.

Reply:
ok. Will change in case of refresh...

Reply:
I had a quick look and there are 601 cases of "() failed" in the code, 125 are using , 111 are using a space, 20 are using : and 189 are using ;.  I wouldn't change this PR but we should agree on a standard and stick with it.  That was only matching () which is the standard suffix to denote function(), this line and others wouldn't appear in that list.